persist-cli-tests/src/test/resources/test-src/input/tool_test_db_push_1/Ballerina.toml

```diff
@@ -2,7 +2,7 @@
 org = "foo"
 name = "tool_test_db_push_1"
 version = "0.1.0"
-distribution = "2201.2.0"
+distribution = "2201.3.0-20221014-074800-bbf5f7a2"
```

Comment:
Lets template this value and update from build script. Or else this needs to be done manually every time

Reply:
```suggestion
distribution = "2201.3.0"
```